OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -98,6 +98,12 @@ public class BuildingCategories
 		[Desc("Delay (in ticks) until rechecking for new BaseProviders.")]
 		public readonly int CheckForNewBasesDelay = 1500;
 
+		[Desc("Delay (in ticks) before deciding which enemy to attack first.")]
+		public readonly int ChooseInitialEnemyDelay = 750;
```

Comment:
I just implemented the intitial delay first and added the interval later. You're right, it's better to keep just the interval.